MiniSim/Menu.swift

```diff
@@ -34,14 +34,18 @@ class Menu: NSMenu {
     }
     
     func getDevices() {
-        if UserDefaults.standard.androidHome == nil {
+        if UserDefaults.standard.enableAndroidEmulators && UserDefaults.standard.androidHome == nil {
```

Comment:
I suppose you can completely remove this check and update line 46 to include `UserDefaults.standard.androidHome == nil`